src/App.tsx

```diff
@@ -5,7 +5,13 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = (movie: Movie) => {
+    if (!movies.some((el:Movie) => el.imdbId === movie.imdbId)) {
+      setMovies([...movies, movie]);
```

Comment:
don't use setter and getter in one place
```suggestion
      setMovies((prev) => [...prev, movie]);
```